rafThrottle.js

```diff
@@ -1,15 +1,18 @@
 const rafThrottle = callback => {
   let requestId
+  let context
 
   const later = args => () => {
     requestId = null
-    callback(...args)
+    callback.apply(context, args)
+    context = null
   }
 
-  const throttled = (...args) => {
+  const throttled = function(...args) {
     if ((requestId === null) || (requestId === undefined)) {
       requestId = requestAnimationFrame(later(args))
     }
+    context = this
```

Comment:
If you do so, you will preserve the context of the first call context instead of the context of the last one. I thought it made more sense to preserve the last. But this is arbitrary, I don't mind changing it.